libs/dal/src/repositories/message/message.entity.ts

```diff
@@ -46,6 +46,8 @@ export class MessageEntity {
     };
   };
 
+  _feedId: string;
```

Comment:
```suggestion
  _feedId?: string;
```

Is there a way the feedId will be missing? if so maybe we should allow null